tests/functional/spec/features/repeating/answer_comparison_repeating_groups.spec.js

```diff
@@ -0,0 +1,31 @@
+const helpers = require('../../../helpers');
+
+const RepeatingComparison1BlockPage = require('../../../pages/features/answer_comparison/repeating_groups/repeating-comparison-1-block.page.js');
+const RepeatingComparison2BlockPage = require('../../../pages/features/answer_comparison/repeating_groups/repeating-comparison-2-block.page.js');
+const SummaryPage = require('../../../pages/features/answer_comparison/repeating_groups/summary.page.js');
+
+describe('Test repeating with answer comparisons', function() {
+
+  it('Given we open the repeating comparison test When we enter different numbers, Then the question should repeat', function() {
+    return helpers.openQuestionnaire('test_repeating_answer_comparison.json').then(() => {
```

Comment:
if you move the `helpers.openQuestionnaire` into a before each then you don't need to repeat it in each test. that applies for all, as its 1 schema per test